inspirehep/modules/workflows/workflows/article.py

```diff
@@ -120,6 +120,20 @@
     ),
 ]
 
+PRE_ENHANCE_RECORD = [
```

Comment:
Better rename this to `SET_AUTO_ACCEPT` or similar, as it does not do anything else. We can rename later if we move more stuff here.

Reply:
or `CHECK_AUTO_ACCEPT`, to be consistent with the other one (`CHECK_IS_UPDATE`)